src/core_plugins/kibana/public/dashboard/grid/dashboard_grid_container.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import { DashboardGrid } from './dashboard_grid';
+import { updatePanel } from '../actions';
+
+const mapStateToProps = ({ dashboardState }, ownProps) => {
+  return {
+    panels: dashboardState.panels,
+    getContainerApi: ownProps.getContainerApi,
+    getEmbeddableHandler: ownProps.getEmbeddableHandler,
+    dashboardViewMode: dashboardState.view.viewMode,
+    hidden: ownProps.hidden
+  };
+};
+
+const mapDispatchToProps = (dispatch) => ({
```

Comment:
You don't need to `dispatch` yourself, you can do:

```
const mapDispatchToProps = {
  onPanelUpdated: updatePanel
}
```

or 

```
const mapDispatchToProps = {
  onPanelUpdated: panel => updatePanel(panel);
}
```

Reply:
Actually, when I use:
`renderEmbeddable: (panelElement) =>renderEmbeddable(embeddableHandler, panelElement, panelId, getContainerApi()),`
the dispatch part does not get called and the code breaks.  When I do:

`renderEmbeddable: (panelElement) => dispatch(renderEmbeddable(embeddableHandler, panelElement, panelId, getContainerApi()))`

It works.  Am I misunderstanding?  Doesn't seem to be automatically wrapping these functions in dispatch...